playbooks/roles/ambari-blueprint/tasks/main.yml

```diff
@@ -94,7 +94,7 @@
     user: "{{ ambari_admin_user }}"
     password: "{{ ambari_admin_password }}"
     headers: '{"X-Requested-By":"ambari"}'
-    body: "{{ cluster_blueprint|to_json }} "
+    body: " {{ cluster_blueprint|to_json }}"
```

Comment:
So actually the space wasn't removed, but just moved from a trailing space to a prefixed one. 
@agriffaut does this change anything? 
and it's not working without space at all?

Reply:
it's not working with some specific edge cases (combinations of ambari and ansible versions), I'd like to keep that space for now unless there's proof it's causing an issue. I'll try ansible 2.5.4 but honestly ansible versions have moved since then so not really a priority. A simple fix would be to set ansible 2.6 as the required version :) https://github.com/hortonworks/ansible-hortonworks#requirements